_posts/2020-07-28-state-of-the-art-nlp-models-from-r/state-of-the-art-nlp-models-from-r.Rmd

```diff
@@ -0,0 +1,283 @@
+---
+title: "State-of-the-art NLP models from R"
+description: |
+  Nowadays, Microsoft, Google, Facebook, and OpenAI share lots of state-of-the-art models in the field of Natural Language Processing. However, there are fewer materials for the practical application of these models for the R community. In this post, we will show how the R users can access and benefit these models as well.
+author:
+  - name: Turgut Abdullayev 
+    url: https://github.com/henry090
+    affiliation: QSS Analytics
+    affiliation_url: http://www.qss.az/
+date: 07-28-2020
+categories:
+  - Natural Language Processing
+creative_commons: CC BY
+repository_url: https://github.com/henry090/transformers
+output: 
+  distill::distill_article:
+    self_contained: false
+    toc: true
+    toc_depth: 2
+preview: files/dino.gif
+---
+
+
+
+<style type="text/css">
+.colab-root {
+    display: inline-block;
+    background: rgba(255, 255, 255, 0.75);
+    padding: 4px 8px;
+    border-radius: 4px;
+    font-size: 11px!important;
+    text-decoration: none;
+    color: #aaa;
+    border: none;
+    font-weight: 300;
+    border: solid 1px rgba(0, 0, 0, 0.08);
+    border-bottom-color: rgba(0, 0, 0, 0.15);
+    text-transform: uppercase;
+    line-height: 16px;
+}
+span.colab-span {
+    background-image: url(https://www.vectorlogo.zone/logos/kaggle/kaggle-ar21.svg);
+    background-repeat: no-repeat;
+    background-size: 51px;
+    background-position-y: -4px;
+    display: inline-block;
+    padding-left: 24px;
+    border-radius: 4px;
+    text-decoration: none;
+}
+</style>
+
+```{r setup, include=FALSE, eval=F,echo=T}
+knitr::opts_chunk$set(echo = FALSE, eval = F, echo = T)
+```
+
+
+## Introduction
+
+Transformers repository from ["Hugging Face"](https://github.com/huggingface/transformers) contains a lot of ready to use state-of-the-art models which is very straightforward to download and fine-tune with Tensorflow & Keras. 
+
+For this purpose the users usually need to get:
+
+* The model itself (e.g. Bert, Albert, RoBerta, GPT-2 and etc.)
+* The tokenizer object
+* The weights of the model
+
+In this post, we will work on classic binary classification task and train our data set on 3 models: 
+
+* [GPT-2](https://blog.openai.com/better-language-models/) from Open AI
+* [RoBERTa](https://arxiv.org/abs/1907.11692) from Facebook
+* [Electra](https://arxiv.org/abs/2003.10555) from Google Research/Stanford University
+
+However, readers should know that one can work with transformers on a variety type down-stream tasks, such as:
```

Comment:
```suggestion
However, readers should know that one can work with transformers on a variety of down-stream tasks, such as:
```

Reply:
I suggest adding a footnote with a few links to tutorials that show how to use these models for downstream tasks.

Reply:
I added hyperlinks